project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject trapperkeeper/lein-template "1.1.0"
+(defproject trapperkeeper/lein-template "1.3.0-SNAPSHOT"
```

Comment:
I arbitrarily chose that to try to make the Y match up with the related version of TK itself.  I'm not attached to that decision and would be happy to change this to whatever other number is preferred if anyone has a different opinion.